### PR TITLE
added support for Black Duck 2019.12.1

### PR DESCRIPTION
**Pull request by msenmurugan**
supports #1200

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1201#" title="Author self-approved">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.12.x/OWNERS)~~ [msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@msenmurugan: once the present PR merges, I will cherry-pick it on top of master in a new PR and assign it to you.

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1201#issuecomment-580856664):

>/cherrypick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by operator-r2d2** [comment]
@msenmurugan: once the present PR merges, I will cherry-pick it on top of saas-2019.12.1 in a new PR and assign it to you.

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1201#issuecomment-580856773):

>/cherrypick saas-2019.12.1


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by yashbhutwala** [comment]
@msenmurugan do we need a PR for the UI's default?

**Comment by yashbhutwala** [comment]
We can have another PR for that, otherwise looks good 😄 

/lgtm

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1201#" title="Author self-approved">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1201#issuecomment-580863715" title="LGTM">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.12.x/OWNERS)~~ [msenmurugan,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/apps/blackduck/latest/containers/imageTags.go [hunk 1]
```diff
@@ -24,6 +24,21 @@ package containers
...
+		"blackduck-jobrunner":      "2019.12.1",
+		"blackduck-registration":   "2019.12.1",
+		"blackduck-scan":           "2019.12.1",
+		"blackduck-webapp":         "2019.12.1",
+		"blackduck-cfssl":          "1.0.1",
+		"blackduck-logstash":       "1.0.5",
+		"blackduck-nginx":          "1.0.14",
+		"blackduck-zookeeper":      "1.0.3",
+		"blackduck-upload-cache":   "1.0.12",
+		"appcheck-worker":          "2019.12",
```
Comment (yashbhutwala) [inline pkg/apps/blackduck/latest/containers/imageTags.go]: curious as to why we are changing a major version of bdba in a dot release of Black Duck

**Comment by operator-r2d2** [comment]
@msenmurugan: new pull request created: #1202

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1201#issuecomment-580856664):

>/cherrypick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by operator-r2d2** [comment]
@msenmurugan: new pull request created: #1203

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1201#issuecomment-580856773):

>/cherrypick saas-2019.12.1


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by operator-r2d2** [comment]
@msenmurugan: base branch (release-2019.12.x) needs to differ from target branch (release-2019.12.x)

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1201#issuecomment-586043794):

>/cherrypick release-2019.12.x


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**